### DISCUSSION TITLE
4.9.0 버전에서 추가된 이전, 다음글 팝업 개선사항

### DISCUSSION THREAD

**Discussion by Cyp9715** [pronist/hELLO#246] [Ideas]
안녕하세요 hELLO 스킨 너무 잘 사용하고 있는 유저입니다.  
다름이 아니라 이번 hELLO 스킨 4.9.0 버전에서 추가된  
* 글에 이전 글/다음 글 팝업이 추가되었습니다.

기능에 대해 개선 아이디어가 있어서 글 남깁니다.

기본적으로 모든 기능은 원할히 잘 작동하나 '이미지가 없는' 포스팅이 이전, 다음 포스팅에 있을경우

<img width="283" alt="image" src="https://github.com/user-attachments/assets/09acb601-19ed-4d58-832c-63f352b3fe88">

이런식으로 많이 보기 흉하게 표시가 되어, 혹시 다음 버전에서 이를 개선해주실 수 있을지 문의 남깁니다.

개인적으로 수정해서 사용할수야 있겠지만, hELLO 스킨 사용자들에게 있어 정식 빌드에서 업데이트 되는것이 보다 유용할것이라 생각합니다.

**Comment by pronist** [comment]
@Cyp9715 안녕하세요.

이 부분은 4.9.0 초기에 배포했다가 늦게 발견한 버그로 제가 의도하지 않은 것이었어서 버전을 별도로 업그레이드하여 릴리즈하지 않고 긴급패치로 다시 배포한 상황입니다. 다시 다운받으시면 해결되어 있을 겁니다.
